codegen-core/common-test-models/constraints.smithy

```diff
@@ -445,6 +445,11 @@ structure ConA {
     maxLengthString: MaxLengthString,
```

Comment:
Do we have a test for a blob shape in the closure of an event stream?

Reply:
Yes we do - https://github.com/awslabs/smithy-rs/pull/2131/files#diff-55153d4b5ba965cb5c4c263983d0b77c0a2ee71ac4a0775e02dc17389e1631faR125